api/src/main/java/org/motechproject/nms/api/web/UserController.java

```diff
@@ -80,17 +93,21 @@ public void setUserLanguage(@PathVariable String serviceName, @RequestBody Langu
 
         // TODO: If no FLW user is found we need to create one here #62
         FrontLineWorker flw = frontLineWorkerService.getByContactNumber(callingNumber);
-        if (null == flw) {
-            throw new NotFoundException(String.format(NOT_FOUND, "callingNumber"));
+        if (flw == null) {
+            throw new NotFoundException(String.format(NOT_FOUND, CALLING_NUMBER));
```

Comment:
I have a test that depends on the values I set in there.  Ticket #62 was
put in to track this todo so no need for the NotImplementedException

When ticket #62 is done then those tests will fail (which is good, whoever
does #62 will then have to fix them for real)

On Thu, Apr 23, 2015 at 9:59 AM, kosh <notifications@github.com> wrote:

> In api/src/main/java/org/motechproject/nms/api/web/UserController.java
> <https://github.com/koshalt/mim/pull/74#discussion_r28981516>:
>
> > @@ -80,17 +93,21 @@ public void setUserLanguage(@PathVariable String serviceName, @RequestBody Langu
> >
> >          // TODO: If no FLW user is found we need to create one here #62
> >          FrontLineWorker flw = frontLineWorkerService.getByContactNumber(callingNumber);
> > -        if (null == flw) {
> > -            throw new NotFoundException(String.format(NOT_FOUND, "callingNumber"));
> > +        if (flw == null) {
> > +            throw new NotFoundException(String.format(NOT_FOUND, CALLING_NUMBER));
>
> Change this to NotImplementedException so that we don't miss it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/koshalt/mim/pull/74/files#r28981516>.
>